wikimedia/wikipedia-ios
# Hide sparkline in top read widget if size category exceeds extra large

**Phabricator:** https://phabricator.wikimedia.org/T323946

### Notes
This PR hides the sparkline and description text present in the large top read widget rows if the user changes their accessibility text size to be larger than extra large. Previously, for large text sizes, the sparkline would overlap with next row text if the user had their device set to larger type sizes. The larger size widget will now more closely resemble the medium sized widget in these cases of larger text sizes.

### Test Steps
1. Add a large top read widget to your home screen
2. Change the accessibility text size in iOS's Settings, confirm the top read widget displays without any elements clipping/overlapping
3. You can also try to enable display zoom in Settings > Display & Brightness > Display and again confirm that no elements are clipping/overlapping

### Screenshots/Videos
![tr_graph](https://user-images.githubusercontent.com/5652327/206561918-2f6688d7-3fbd-4a3f-8e1a-db9dfa341396.jpg)
